thiagospassos/tryghost
# Upgrade to 1.22.1 (#1)

* initial

* allows config json

* add postinstall script

* new postinstall script

* add old version of knex-migrator on devDep

* use old version of knex

* use prepend-node-path

* set npm config on preinstall

* previous

* revert to last previous success script

* unwanted shrinkwrap

* fix postinstall

* remove grunt in postinstall

* use npm bin for postinstall

* revert to fix postinstall

* move database setting to migratorconfig

* postinstall

* revert postinstall

* put mgpath

* latest node

* new knex migrator approach

* use older version of node

* test specific node request

* update all packages

* try npm bin version

* typo fix

* retry

* remove iinode, use more specific engines

* use yarn instead of npm

* try to use install script

* move sqlite to deps

* change in deployment

* remove yarn.lock

* this should work

* remove npm install from post install as the knex-migrator is not getting called

* upgrade to 1.22.1

* set default websiteUrl

* upload pre-built assets
